spacy_llm/tests/tasks/test_spancat.py

```
@@ -0,0 +1,464 @@
+# mypy: ignore-errors
```

Did you mean to leave this long-term?

---

Ideally we don't ignore errors file-wide.

---

I copy-pasted the test suite for the NER. I'll change that in both files.